### PR TITLE
Expanding examples on #514

### PR DESCRIPTION
**Pull request by j0rt3g4**
Cmdlet doesn't work on already created disks.
Expanding examples

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Solving Issue #514

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/storage/get-physicalextent.md [hunk 1]
```diff
@@ -21,22 +21,22 @@ ms.assetid: 65DD9798-9AF5-4E64-AA66-421AA98F8704
 # Get-PhysicalExtent
 
 ## SYNOPSIS
-Gets physical allocations for a physical disk, storage tier, or virtual disk.
+This cmdlet gets physical allocations for a physical disk, storage tier, or virtual disk. An "extent" is the area of space on a pooled disk that contains one fragment of the data for storage space,  also can be synonymous of "allocation" or as a "slab."
```
Comment (JohanFreelancer9) [inline docset/windows/storage/get-physicalextent.md]:
```suggestion
This cmdlet gets physical allocations for a physical disk, storage tier, or virtual disk. An "extent" is the area of space on a pooled disk that contains one fragment of the data for storage space,  and can also be synonymous with "allocation" or "slab."
```

File: docset/windows/storage/get-physicalextent.md [hunk 2]
```diff
@@ -21,22 +21,22 @@ ms.assetid: 65DD9798-9AF5-4E64-AA66-421AA98F8704
 # Get-PhysicalExtent
 
 ## SYNOPSIS
-Gets physical allocations for a physical disk, storage tier, or virtual disk.
+This cmdlet gets physical allocations for a physical disk, storage tier, or virtual disk. An "extent" is the area of space on a pooled disk that contains one fragment of the data for storage space,  and can also be synonymous with "allocation" or "slab".
```
Comment (nenonix) [inline docset/windows/storage/get-physicalextent.md]:
```suggestion
This cmdlet gets physical allocations for a physical disk, storage tier, or virtual disk. The "extent" (also known as "allocation" or "slab)  is the area on a pooled disk containing one fragment of data for storage space,
```

File: docset/windows/storage/get-physicalextent.md [hunk 3]
```diff
@@ -45,14 +45,29 @@ The **Get-PhysicalExtent** cmdlet gets the physical allocations for a physical d
...
 
 This command gets all physical extents on the physical disk named PhysicalDisk4.
 The command uses **Get-PhysicalDisk** to obtain PhysicalDisk4.
 
+### Example 3: Get all physical extents on a disk (Other Version)
+```Powershell
+PS C:\>Get-PhysicalDisk -FriendlyName "PhysicalDisk4" | Get-PhysicalExtent
+```
+
+This command gets all physical extents on the physical disk named PhysicalDisk4, using powershell pipe.
```
Comment (nenonix) [inline docset/windows/storage/get-physicalextent.md]:
```suggestion
This command gets all physical extents on the physical disk named PhysicalDisk4, using PowerShell pipe.
```

**Comment by j0rt3g4** [comment]
@nenonix  I don't understand I've added the changed. what is missing now?

**Comment by nenonix** [comment]
you were so quick...you pushed my edits before i left review comment, 😄

**Comment by j0rt3g4** [comment]
Haha yeah :)

**Pull request merged**